src/index.js

```diff
@@ -105,14 +131,66 @@ const addListToPages = () => {
 
     wrap.setAttribute('class', 'flex');
     container.setAttribute('class', 'listing flex');
-    des.innerHTML = sorted[x].description;
+    des.innerHTML = pages[x].description;
   }
+
+  const button = document.createElement('button');
+  section.append(button);
+  button.setAttribute('class', 'clear');
+  button.setAttribute('type', 'button');
+  button.innerHTML = 'Clear all Completed';
+  count += 1;
 };
 
-addListToPages();
+if (count === 0) addListToPages(to_Do_List.Lists);
+
+function populate() {
+  const instances = {
+    figures: [],
+  };
+  instances.figures.push(to_Do_List);
+  localStorage.setItem('instances', JSON.stringify(instances));
+}
+
+const box = document.querySelectorAll('.checker');
+box.forEach((element) => {
+  element.addEventListener('change', (e) => {
+    const parent = e.target.parentNode;
+    const des = parent.querySelector('p');
+    const content = des.innerHTML;
+    const ans = to_Do_List.returnDes(content);
+    if (e.target.checked) {
+      to_Do_List.changeComplete(ans);
+      populate();
+    } else {
+      to_Do_List.changeComplete(ans);
+      populate();
+    }
+  });
+});
+
+function setForm() {
+  const restore = JSON.parse(localStorage.getItem('instances'));
```

Comment:
- [ ] I would recommend using the ES6 syntax of javascript. As it makes the code look cleaner and it's also recommended in the curriculum.